DEVELOPMENT.md

```diff
@@ -0,0 +1,20 @@
+Developer docs
+==================================
+
+## 📝 Prerequisites
+* ffmpeg
+* Python (Current required version van be found in [pyproject.toml](https://github.com/music-assistant/server/blob/main/pyproject.toml#LL11C14-L11C14))
```

Comment:
Python 3.11 is minimal required (or check the pyproject for current required version)